google-map-marker.html

```diff
@@ -396,6 +396,7 @@
           this.info.open(this.map, this.marker);
           this.fire('google-map-marker-open');
         } else {
+          this.content = this.info.getContent();
```

Comment:
When you use the DOM node instead of a copy of the HTML, the info window moves the node and is no longer under this element. When the InfoWindow is closed, you have to ensure that the contents are stored here, otherwise they're overwritten and lost whenever the InfoWindow is used to display some other marker's content.